src/main/java/org/folio/dew/batch/acquisitions/edifact/jobs/SaveToFileStorageTasklet.java

```diff
@@ -28,30 +30,32 @@
 @StepScope
 @Log4j2
 public class SaveToFileStorageTasklet implements Tasklet {
-  private final ObjectMapper ediObjectMapper;
-  private final FTPStorageService ftpStorageService;
-
-  private final RemoteFilesStorage remoteFilesStorage;
 
   private static final String SFTP_PROTOCOL = "sftp://";
-  @Value("#{jobParameters['edifactFileName']}")
-  private String edifactFileName;
+
+  private final ObjectMapper ediObjectMapper;
+  private final FTPStorageService ftpStorageService;
 
   @Override
   @SneakyThrows
   public RepeatStatus execute(StepContribution contribution, ChunkContext chunkContext) {
-    var stepExecution = chunkContext.getStepContext().getStepExecution();
     var jobParameters = chunkContext.getStepContext().getJobParameters();
-    var ediExportConfig = ediObjectMapper.readValue((String)jobParameters.get(EDIFACT_ORDERS_EXPORT), VendorEdiOrdersExportConfig.class);
-    var uploadedFilePath = (String) ExecutionContextUtils.getExecutionVariable(stepExecution, UPLOADED_FILE_PATH);
+    var ediExportConfig = ediObjectMapper.readValue((String) jobParameters.get(EDIFACT_ORDERS_EXPORT), VendorEdiOrdersExportConfig.class);
+    if (ediExportConfig.getIntegrationType() != ORDERING && ediExportConfig.getTransmissionMethod() != FTP) {
+      log.info("execute:: Transmission method is not FTP, skipping the step");
+      return RepeatStatus.FINISHED;
+    }
 
     String host = ediExportConfig.getEdiFtp().getServerAddress().replace(SFTP_PROTOCOL, "");
     // skip ftp upload if address not specified
     if (StringUtils.isEmpty(host)) {
       return RepeatStatus.FINISHED;
     }
-    byte[] fileContent = remoteFilesStorage.readAllBytes(uploadedFilePath);
-    ftpStorageService.uploadToFtp(ediExportConfig, fileContent, FilenameUtils.getName(uploadedFilePath));
+
+    var stepExecution = chunkContext.getStepContext().getStepExecution();
+    var fileName = (String) ExecutionContextUtils.getExecutionVariable(stepExecution, EDIFACT_FILE_NAME);
```

Comment:
can we rename to ACQ_EXPORT_FILE_NAME? In this case we will have the same convention as for ACQ_EXPORT_FILE